robottelo/api/entity_fixtures.py

```diff
@@ -46,9 +46,8 @@ def module_smart_proxy(module_location):
 
 
 @pytest.fixture(scope='module')
-def module_domain(module_org, module_location, module_smart_proxy):
+def module_domain(module_org, module_location):
```

Comment:
removed `module_smart_proxy` because :
- Domain is not managed by Satellite DNS and if  specify that mean `DNS Capsule to use within this domain for managing A records`
- Avoid tests fails with error `Unable to set DNS entry ([RestClient::BadRequest]: 400 Bad Request) for Capsule`